docs/build_process.rst

```diff
@@ -6,40 +6,29 @@ Understanding the Build Process
 Logging
 -------
 
-Logs from builds is made available via the orchestrator build,
+Logs from builds is made available via osbs-client API,
 and clients (including koji-containerbuild) are able to separate
 individual build logs out from that log stream using an
```

Comment:
It might be clearer if we don't use the word "build" for two different meanings. How about "Logs from builds", "individual task logs"?

Reply:
updated.